l10n_nl_partner_name/model/__init__.py

```diff
@@ -0,0 +1,2 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
```

Comment:
And model directory can be renamed `models` (use `git mv' to prevent bogus deleted and new files).